frontend/src/app/testResults/TestResultsList.test.tsx

```diff
@@ -43,6 +46,12 @@ const store = mockStore({
   facility: { id: "1", name: "Facility 1" },
 });
 
+const testResults = {
```

Comment:
defining the GQL response shape here rather than in the mock file since we're testing both component-level and GQL-boundary-level interactions that need the same data but in different shapes.